### DISCUSSION TITLE
给 mirror.xyz 添加订阅功能

### DISCUSSION THREAD

**Discussion by luckyyang** [rebase-network/hello-world#2] [Event]
https://app.submirror.xyz/

将一篇 mirror.xyz 的文章输入这个网站，就能生成不同的订阅链接，支持 RSS、Feedly、Inoreader 和 Telegram bot

**Comment by luckyyang** [comment]
其实将链接中的 mirror.xyz 改为 submirror.xyz 即可支持 RSS 订阅

**Reply by zhangtaotaoBTC** [reply]
我试了，为啥显示成乱码了，感觉还是体验不好，不好用。
![捕获](https://user-images.githubusercontent.com/68153935/156712013-3aee6fb7-99a8-45c8-aa39-59e2229a0726.JPG)

**Reply by luckyyang** [reply]
不是你这么用的，你得 copy url，然后 paste 到你的 rss 客户端中，比如 reeder：
![2022-03-04 at 3 27 PM](https://user-images.githubusercontent.com/1963646/156718656-c908873a-fc5e-437a-abef-e47602320cef.png)

**Comment by johnsonjie** [comment]
niubility
